README.md

```diff
@@ -297,7 +297,7 @@ npm install
 npm run start
 ```
 
-Open http://localhost:5000 in your browser to try the demo.
+Open `http://localhost:5000` in your browser to try the demo.
```

Comment:
should we actually link them [http://localhost:5000](http://localhost:5000) and then add a next line disable?

Reply:
I think we went with this model in other repos. Let's keep doing this.